service/src/main/java/uk/gov/hmcts/reform/fpl/model/common/EmailAddress.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.fpl.model.common;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class EmailAddress {
+    private final String email;
+
+    @JsonCreator
+    private EmailAddress(@JsonProperty("email") String email) {
```

Comment:
JSON constructor should not be needed. Please remove since field name is same as JSON property.

Reply:
it is, for some reason it throws an error when we remove this saying the mapping isnt correct, I am pretty sure James and possible bridget had similar error. This was the only way to fix.

Reply:
You should be able to remove. @JsonProperty is needed when variable names don't match the json field you want to map to the variable

Reply:
MappingException?

Reply:
Please see https://github.com/FasterXML/jackson-module-parameter-names/issues/21

Reply:
Was able to remove mine Becky. You might be seeing a lombok issue locally.

Reply:
I cant remove it when i do and try and create applicant i get 

threw exception [Request processing failed; nested exception is uk.gov.hmcts.reform.fpl.exceptions.MappingException: com.fasterxml.jackson.databind.exc.MismatchedInputException: Cannot construct instance of `uk.gov.hmcts.reform.fpl.model.common.EmailAddress` (although at least one Creator exists): cannot deserialize from Object value (no delegate- or property-based Creator)

Reply:
https://github.com/rzwitserloot/lombok/issues/1912